index.js

```diff
@@ -62,7 +62,7 @@ module.exports = class MongooseTrailpack extends DatastoreTrailpack {
         schema.statics = model.statics
         schema.methods = model.methods
 
-        model.onSchema(schema)
+        model.onSchema(schema, this.app)
```

Comment:
Hello! Any news on this?
Can I proceed with the edit moving onSchema into config method?

I'm quite stuck for jaumard/trailpack-passport#77 without progress on this
@konstantinzolotarev @jaumard